data/strings/strings.txt

```diff
@@ -5687,6 +5687,53 @@
     zh-Hans = 测试语音指令（TTS，Text-To-Speech）
     zh-Hant = 測試語音指令（TTS、文本轉語音
 
+  [pref_tts_open_system_settings]
+    comment = button to open the system's TTS setting menu
+    tags = android
+    en = Open system TTS settings
+    af = Maak stelsel TTS-instellings oop
+    ar = افتح إعدادات تحويل النص إلى كلام (TTS) للنظام
+    az = Sistem TTS parametrlərini açın
+    be = Адкрыйце сістэмныя налады TTS
+    bg = Настройки на отворената система TTS
+    ca = Obriu la configuració de TTS del sistema
+    cs = Nastavení TTS otevřeného systému
+    da = Indstillinger for åbent system TTS
+    de = TTS-Einstellungen für das offene System
+    el = Ρυθμίσεις TTS ανοικτού συστήματος
+    es = Ajustes TTS del sistema abierto
+    es-MX = Abrir la configuración TTS del sistema
+    et = Avatud süsteemi TTS seaded
+    eu = Ireki sistemaren TTS ezarpenak
+    fa = تنظیمات TTS سیستم را باز کنید
+    fi = Avoimen järjestelmän TTS-asetukset
+    fr = Ouvrir les paramètres de synthèse vocale
+    he = פתח את הגדרות ה-TTS של המערכת
+    hi = सिस्टम टीटीएस सेटिंग्स खोलें
+    hu = Nyitott rendszer TTS beállítások
+    id = Membuka pengaturan TTS sistem
+    it = Impostazioni TTS del sistema aperto
+    ja = オープンシステムのTTS設定
+    ko = 시스템 TTS 설정 열기
+    lt = Atviros sistemos TTS nustatymai
+    mr = सिस्टम TTS सेटिंग्ज उघडा
+    nb = Åpne TTS-innstillinger for systemet
+    nl = TTS-instellingen open systeem
+    pl = Ustawienia otwartego systemu TTS
+    pt = Abrir as definições de TTS do sistema
+    pt-BR = Configurações de TTS do sistema aberto
+    ro = Setări TTS pentru sistemul deschis
```

Comment:
```suggestion
    ro = Deschide setările TTS de sistem
```

But please note that I'm not an Android user. This, however, is definitely more correct than the auto translation.